package.json

```diff
@@ -66,7 +66,7 @@
     "@kvaster/zwavejs-prom": "^0.0.3",
     "@mdi/js": "7.4.47",
     "@zwave-js/log-transport-json": "^3.0.0",
-    "@zwave-js/server": "^1.40.2",
+    "@zwave-js/server": "zwave-js/zwave-js-server#a10f638ea05756809884777d1c397b847b4ab293",
```

Comment:
@AlCalzone do we have a zjs server release?

Reply:
not yet. waiting on a review